coffeecard/CoffeeCard.Models/DataTransferObjects/v2/Product/AddProductRequest.cs

```diff
@@ -0,0 +1,26 @@
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace CoffeeCard.Models.DataTransferObjects.v2.Product
+{
+    /// <summary>
+    /// Initiate a new product add request
+    /// </summary>
+    public class AddProductRequest
+    {
+        [Required]
+        public int Price { get; set; }
+
+        [Required]
+        public int NumberOfTickets { get; set; }
+
+        [Required]
+        public string Name { get; set; }
```

Comment:
Setting Required just ensures that the property is present in the JSON request, not that is has a value. We should add a property to ensure that empty strings are considered invalid. Similarly for Description